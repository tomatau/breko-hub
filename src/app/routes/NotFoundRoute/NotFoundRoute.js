import React from 'react'
import { Helmet as DocumentMeta } from 'react-helmet-async'
import { notFoundRoute as notFoundRouteCopy } from 'app/copy'

export default class NotFoundRoute extends React.Component {
  render() {
    return (
      <>
        <DocumentMeta>
          <title>{notFoundRouteCopy.documentTitle}</title>
        </DocumentMeta>
        <section className='NotFoundRoute'>
          {notFoundRouteCopy.content}
        </section>
      </>
    )
  }
}
