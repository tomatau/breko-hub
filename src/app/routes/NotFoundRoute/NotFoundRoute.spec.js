import React from 'react'
import NotFoundRoute from './NotFoundRoute'

describe(`NotFoundRoute Component`, function () {
  helpers.setupSnapshots(__filename)

  it(`matches snapshot`, () => {
    expect(
      snap(shallow(<NotFoundRoute />))
    ).to.matchSnapshot()
  })
})
