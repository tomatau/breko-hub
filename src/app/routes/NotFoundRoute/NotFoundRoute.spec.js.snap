// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundRoute Component matches snapshot 1`] = `
<Fragment>
  <Helmet
    defer={false}
    encodeSpecialCharacters={true}
  >
    <title>
      Doesn't exist
    </title>
  </Helmet>
  <section
    className="NotFoundRoute"
  >
    <h2>
      404
    </h2>
    <p>
      Page not found!
    </p>
  </section>
</Fragment>
`;
